en/identity-server/next/docs/apis/organization-apis/restapis/organization-user-share.yaml

```
@@ -316,7 +323,7 @@ paths:
         - User Accessible Organizations
       summary: Get organizations a user has access to
       description: |
-        This API retrieves the list of organizations where the specified user has shared access, with support for pagination and filtering.
+        This API retrieves the list of organizations where the specified user has shared access. Supports pagination and filtering. 
```

```suggestion
        This API retrieves the list of organizations where the specified user has shared access.
```
Let's remove pagination support.

---

Verified.